test_memory_diff/test_diff.py

```diff
@@ -5,8 +5,9 @@
 class Test_TestIncrementDecrement(unittest.TestCase):
 
     def test_saving_diff(self):
-        diff_object = Diff('/mnt/c/Users/Antonio/Desktop/memory-diff/test_memory_diff/fileNew.txt', '/mnt/c/Users/Antonio/Desktop/memory-diff/test_memory_diff/fileOld.txt')
-        modified_file = diff_object.saving_diff()
+        diff_object = Diff('/mnt/c/Users/Antonio/Desktop/memory-diff/test_memory_diff/Matecat_New.tmx', '/mnt/c/Users/Antonio/Desktop/memory-diff/test_memory_diff/Matecat_Old.tmx','/mnt/c/Users/Antonio/Desktop/memory-diff/test_memory_diff/diff_file.tmx')
```

Comment:
put the path as input of the test function, is not a best practice to hardcode paths

Reply:
I don't understand what i've to do

Reply:
instead of put the file path inside the code you should take the file path as input